containers/forwarder/Dockerfile

```diff
@@ -24,11 +22,14 @@ RUN export VERSION=$(cat /.splunk-version) && export VERSION_HASH=$(cat /.splunk
     mkdir -p /opt/splunkforwarder/etc/apps/osd_monitored_logs/{local,metadata} && \
     mkdir -p /opt/splunkforwarder/etc/apps/splunkauth/{default,local,metadata} && \
     chown -R splunk:splunk /opt/splunkforwarder && \
-    chgrp splunk /runner && \
-    chmod +x /runner /bin/systemctl && \
     sed -i 's/\.maxBackupIndex=5/\.maxBackupIndex=1/g' /opt/splunkforwarder/etc/log.cfg && \
     sed -i 's/\.maxFileSize=25000000/\.maxFileSize=250000/g' /opt/splunkforwarder/etc/log.cfg
 
+ADD containers/forwarder/bin/systemctl_wrapper.sh /bin/systemctl
```

Comment:
We should only use `ADD` instructions if there's a good reason to do so. This can just be a `COPY`. The rest of the Dockerfile can be dealt with later.